Makefile

```diff
@@ -1,17 +1,19 @@
-# Pipelines Project Make File
 # author: Sylvain Anderegg
 
 PY_FILES = $(strip $(shell find services -iname '*.py'))
 
 build:
 	docker-compose -f services/docker-compose.yml build --no-cache
```

Comment:
Why the --no-cache flag in the case of build? for rebuild I agree

Reply:
I propose we have 3 possible builds:
- dev
- test or production
- deploy
then the following recipies:
- up (calls docker swarm init, docker-compose -f ... up --build)
- down (calls docker-compose -f ... down, docker swarm leave -f)